java/pom.xml

```diff
@@ -46,6 +46,13 @@
     </dependencyManagement>
 
     <dependencies>
+        <dependency>
+            <groupId>com.eclipsesource.minimal-json</groupId>
+            <artifactId>minimal-json</artifactId>
+            <version>0.9.5</version>
+            <scope>provided</scope>
```

Comment:
If shaded probably not what youiintended.

Reply:
Yeah, realised I had to remove that